### DISCUSSION TITLE
Possibility to disable 'Delete' in menu

### DISCUSSION THREAD

**Discussion by oherau** [OnionUI/Onion#1529] [Ideas]
When browsing games, the [Select] button gives access to a context menu.
Just below the option "Add to favorites" is the "Delete" button.
It is not very safe as I can delete a game by mistake.

Would it be possible to add a switch in Tweaks to disable the "Delete" option from the game contextual menu ?

**Comment by gitleecher** [comment]
The correct path would be to add a "Are you sure?" context popup if there isn't one.

**Reply by Aemiii91** [reply]
There is.

**Reply by oherau** [reply]
Ok. I think the confirmation popup is enough to prevent any mistake.
